setup.py

```diff
@@ -12,7 +12,7 @@
 min_numpy_build_version = "1.11"
 min_numpy_run_version = "1.15"
 min_llvmlite_version = "0.31.0.dev0"
-max_llvmlite_version = "0.32.0.dev0"
+max_llvmlite_version = "0.33.0.dev0"
```

Comment:
Wondering if this should be `"0.33.0.rc*"`? Or perhaps move to that later in the release cycle.

Reply:
My reasoning would be that we don't need to include the RC until we have it, but it's hard to attack this problem with reason, seems like only trial and error will get the job done.